ERROR - 2024-06-27 14:18:46 --> Error connecting to the database: mysqli_sql_exception: Host 'localhost' is not allowed to connect to this MariaDB server in C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'db_ci4', 3306, '', 0)
#1 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\CI4\app\Controllers\AuthController.php(31): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\CodeIgniter.php(941): App\Controllers\AuthController->login()
#8 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CI4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Host 'localhost' is not allowed to connect to this MariaDB server in C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\CI4\app\Controllers\AuthController.php(31): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\CodeIgniter.php(941): App\Controllers\AuthController->login()
#7 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#8 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\CI4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2024-06-27 14:18:46 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Host 'localhost' is not allowed to connect to this MariaDB server
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\AuthController.php(31): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\AuthController->login()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\CI4\\public\\index.php')
ERROR - 2024-06-27 14:20:14 --> Error connecting to the database: mysqli_sql_exception: Host 'localhost' is not allowed to connect to this MariaDB server in C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'db_ci4', 3306, '', 0)
#1 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\CI4\app\Controllers\AuthController.php(31): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\CodeIgniter.php(941): App\Controllers\AuthController->login()
#8 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CI4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Host 'localhost' is not allowed to connect to this MariaDB server in C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\CI4\app\Controllers\AuthController.php(31): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\CodeIgniter.php(941): App\Controllers\AuthController->login()
#7 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#8 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\CI4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\CI4\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2024-06-27 14:20:14 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Host 'localhost' is not allowed to connect to this MariaDB server
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\AuthController.php(31): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\AuthController->login()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\CI4\\public\\index.php')
